alpha/lib/model/index/entryIndex.php

```diff
@@ -378,6 +378,64 @@ public static function getSphinxConditionsToKeep()
 		return self::$conditionToKeep;
 	}
 
+	protected static $apiCompareAttributesMap;
+
+	public static function getApiCompareAttributesMap()
+	{
+		if (!self::$apiCompareAttributesMap)
+		{
+			self::$apiCompareAttributesMap = array(
+				'status' => 'entry_status',
```

Comment:
Roman,
Can't we add these definitions to the index XML?

Eran,
We need to add verification to the static code validator, could you remind us where it could be found?